internal/provider/resource_variable.go

```diff
@@ -94,7 +94,7 @@ func (t *variableResource) GetSchema(ctx context.Context) (tfsdk.Schema, diag.Di
 				Type:                types.StringType,
 				MarkdownDescription: "This variable's value.",
 				Description:         "This variable's value.",
-				Optional:            true,
+				Required:            true,
```

Comment:
Since this was changed, you will need to regenerate the docs.

Reply:
ah good catch!

Reply:
updated